tests/e2e/registry/test_registry_pod_respin.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+import logging
+
+
+from ocs_ci.ocs.constants import OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.registry import (
+    validate_registry_pod_status,
+    image_pull, image_push, image_list_all, image_rm,
+    check_image_exists_in_registry
+)
+from ocs_ci.framework.testlib import E2ETest, workloads
+from ocs_ci.utility.svt import svt_setup, svt_cleanup
+from tests import disruption_helpers
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+IMAGE_URL = 'docker.io/library/busybox'
+
+
+@workloads
+class TestRegistryPodRespin(E2ETest):
+    """
+    Test to run svt workload for pushing
+    images to registry and with Ceph pods respin
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
```

Comment:
you can add (line 35)
self.image_path = None
If you reach the tear down stage before you reach line number 84 there will be an error.

Reply:
agreed, added at line 42